pymatgen/analysis/defects/point_defects.py

```diff
@@ -1530,8 +1530,9 @@ class StructureMotifInterstitial(Defect):
     __supported_types = ("tet", "oct", "bcc", "tetoct")
 
     def __init__(self, structure, inter_elem,
-                 motif_types=["tet", "oct", "tetoct"],
-                 op_targets=[1.0, 1.0, 1.0], op_threshs=[0.5, 0.5, 0.5],
+                 motif_types=None,
```

Comment:
I got the impression that tuples are not as widely used as using None (for default lists).  They make sense in this context, yes.  But I mainly found online references that would suggest to use None and then set up the list in the __init__.  But if you approve usage of tuples, I'll stick to using tuples in these situations.